infrateam_first_responder.md

```diff
@@ -37,85 +35,85 @@ PRs to this doc are always welcome, but especially while we're ironing things ou
-* https://github.com/sul-dlss/DevOpsDocs - e.g. what an ops or devops person would need to know to handle the situation
+If you need to triage or troubleshoot a problem and realize some documentation is missing, please provide it. List of appropriate places:
+* https://github.com/sul-dlss/DevOpsDocs - e.g., what an ops or devops person would need to know to handle the situation
 * README or other top level markdown doc in codebase (viewable via github)
   * Which codebase would need to be apparent from the problem
 * Wiki for the codebase
-* ? - in general, just try to consider where the person interested in the info might look first.  A user might go to the wiki, a dev might go to the README, ops folks might head to DevOpsDocs.  Use your best judgement and ask for feedback if unsure.
+* ? - in general, just try to consider where the person interested in the info might look first. A user might go to the wiki, a dev might go to the README, ops folks might head to DevOpsDocs. Use your best judgement and ask for feedback if unsure.
 
-The above documents should be useful and current.  Please submit improvements as PRs for review.
+The above documents should be useful and current. Please submit improvements as PRs for review.
 
 If for some reason documentation is a significant undertaking, the call for documentation can be filed as an issue and prioritized/resourced by management.
 
-## Improve First Responder Instructions as Needed
+### Improve First Responder Instructions as Needed
 
 We need this document to be useful and current.  Please submit improvements as PRs for review.
 
-## Other Duties as Assigned
+### Other Duties as Assigned
 
 * Management may choose to have the first responder handle a non-project work ticket
   * If so, ensure you assign the ticket to yourself and put it in the "in progress" column the project board https://github.com/orgs/sul-dlss/projects/1
 * First responder may be asked to spearhead a work estimate https://github.com/sul-dlss-labs/estimation (note that these are, by definition, meant to be done by more than one person; if it's smaller, should it be a ticket in a project?)
 
-# How to Proactively Check for Production Problems
+## How to Proactively Check for Production Problems
 
-At the very least, the first responder should be watching incoming alerts from Honeybadger.  (https://app.honeybadger.io/projects)
+At the very least, the first responder should be watching:
 
-Other places to monitor:
-* Nagios alerts (https://sul-nagios-prod.stanford.edu/nagios/  The Services tab on the left column will give overview of checks.  The Problems->Services tab will give you what’s alerting.)
-* feedback email lists, e.g. argo-feedback, sinopia-feedback, etc
+* [Honeybadger alerts](https://app.honeybadger.io/projects) for projects in the Infrastructure portfolio
+  * **Note** that you may want to clear old alerts at the beginning of your first responder shift so you have a clean slate to monitor.
```

Comment:
"mark old alerts as resolved" rather than "clear old alerts"?

Reply:
went with "resolve old alerts in the Honeybadger UI" 

thanks!